Description: Use XDG_DOCUMENTS_DIR as default for 'Print to File'
 This patch uses $XDG_DOCUMENTS_DIR as the default directory rather
 than the home directory when the user selects Print to File in the
 GtkPrinterOptionWidget.
Author: Bilal Akhtar <bilalakhtar@ubuntu.com>
Bug-Ubuntu: https://launchpad.net/bugs/611011
Index: gtk+-2.24.12/gtk/gtkprinteroptionwidget.c
===================================================================
--- gtk+-2.24.12.orig/gtk/gtkprinteroptionwidget.c	2012-01-11 13:06:05.000000000 +1300
+++ gtk+-2.24.12/gtk/gtkprinteroptionwidget.c	2012-09-04 16:45:23.506558688 +1200
@@ -824,6 +824,8 @@
         priv->entry = gtk_entry_new ();
         priv->combo = gtk_file_chooser_button_new (source->display_text,
                                                    GTK_FILE_CHOOSER_ACTION_SELECT_FOLDER);
+        gtk_file_chooser_set_current_folder (GTK_FILE_CHOOSER (priv->combo),
+                                                   g_get_user_special_dir (G_USER_DIRECTORY_DOCUMENTS));
 
         g_object_set (priv->combo, "local-only", FALSE, NULL);
         gtk_entry_set_activates_default (GTK_ENTRY (priv->entry),
